pkg/cmd/server/api/v1/types.go

```diff
@@ -97,6 +100,11 @@ type MasterConfig struct {
 	// to wait until a notification to the server is received before launching them.
 	PauseControllers bool `json:"pauseControllers,omitempty"`
 
+	// Allow to disable OpenShift components. Don't let it appear in config if
```

Comment:
Godoc format
```
// DisabledFeatures is a list of features that should not be started.  We omitempty here because its very unlikely that anyone will want to manually disable features and we don't want to encourage it.
```